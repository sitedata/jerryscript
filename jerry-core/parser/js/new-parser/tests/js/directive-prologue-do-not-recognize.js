/* This test contains directive prologue parser tests. */

/* in this case sould not be treated as a directive prologue */
"use strict"
+ " another time"
