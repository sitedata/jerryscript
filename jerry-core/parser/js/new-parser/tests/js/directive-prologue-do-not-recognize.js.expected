
--- Script parsing start ---

  [  1] CBC_ADD_TWO_LITERALS idx:0->string(use strict) idx:1->string( another time)
  [  0] CBC_POP_BLOCK

Final byte code dump:

  Maximum stack depth: 1
  Flags: [small_lit_enc]
  Argument range end: 0
  Register range end: 0
  Identifier range end: 0
  Const literal range end: 2
  Literal range end: 2

   0 : CBC_ADD_TWO_LITERALS idx:0(lit)->string(use strict) idx:1(lit)->string( another time)
   3 : CBC_POP_BLOCK
   4 : CBC_RETURN_WITH_BLOCK

Byte code size: 5 bytes

Script parsing successfully completed. Total byte code size: 5 bytes

--- Script parsing end ---

